content/articles/web-automation-with-selenium/index.md

```diff
@@ -0,0 +1,107 @@
+### Introduction
+Automation can be seen as a process of removing human effort in a process. It can be seen as a process of using electronic machines or robots to perform tasks. Automation can be done in many processes where humans are involved but in this article, we will be looking at automating web processes. 
+The ability to enable software robots to automatically perform processes and tasks on the web is known as WEB AUTOMATION. These processes may include; searching the web, deleting emails, filling forms, logging in to a website, and many other processes. The need for speed in performing tasks is a necessity in our modern world, this makes the automation of repetitive tasks necessary.
+Selenium is a framework for web application testing. In python, Selenium can be seen as a set of libraries that helps developers interact with the web to enable the automation of web processes. Primarily Selenium is used for automating software tests but it is also used for web scraping. Selenium is a very powerful tool when it comes to interacting with web browsers, it works with all modern web browsers and its scripts can be written in different programming languages such as Java, python, c#, etc.
+In this guide, we will be looking at how to use selenium to write scripts that will automate basic web tasks using python to give you a background of how Selenium works.
+### Prerequisite
+To understand this guide the reader must be familiar with HTML syntax such as; tags, elements, Ids, and classes. It is also assumed the reader is familiar with the basics of python programming language, concepts such as its syntax and creating a virtual environment are required.
+### Goal
+In this guide, we will be writing a python script that will perform a search on a browser. At the end of this guide the reader will be able to write a python script that can;
```

Comment:
Perform a search in the browser?

Do you mean to perform a Google search on something? If so, please state that here. What exactly would be the result?